pandas/tests/tslibs/test_conversion.py

```diff
@@ -57,9 +57,10 @@ def test_tz_convert_single_matches_tz_convert(tz_aware_fixture, freq):
     ],
 )
 def test_tz_convert_corner(arr):
-    result = tzconversion.tz_convert(
-        arr, timezones.maybe_get_tz("US/Eastern"), timezones.maybe_get_tz("Asia/Tokyo")
-    )
```

Comment:
is this test no longer relevant?

Reply:
this PR requires that one of the two args be UTC

Reply:
hmm, that's what tz_convert_single was doing before? so what's the difference now between tz_convert and tz_convert_single? and is converting between 2 tzs no longer needed?

Reply:
> so what's the difference now between tz_convert and tz_convert_single?

One handles a scalar (i.e. Timestamp) the other handles an array.  I'm trying to de-duplicate these further, havent found a way to do so without a perf hit.

> and is converting between 2 tzs no longer needed?

correct.